src/charm.py

```diff
@@ -462,7 +462,7 @@ def _make_nginx_config(self, env_config: dict) -> str:
         Returns:
             A fully configured NGINX conf file
         """
-        with open("templates/nginx_cfg.tmpl", "r", encoding="utf-8") as file:
+        with open("rockfile/nginx_cfg.tmpl", "r", encoding="utf-8") as file:
```

Comment:
For directory structures, I've seen other repositories have files inside the `<charm-name>_rock` directory for files that need to be baked into the image and `templates` directory contains the templates that need to be dynamically created. Would this be applicable?

Reply:
Yes it is, I'll make the respective changes